behaviors/wysiwyg.js

```diff
@@ -14,6 +14,17 @@ var _ = require('lodash'),
   site = require('../services/site'),
   refAttr = references.referenceAttribute;
 
+// pass config actions to text-model
+model.updateSameAs({
```

Comment:
Stuff like this makes me miss python
`{"H%s"%(index): "STRONG" for index in range(1,7)}`